packages/indexer/Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM rust:1.76-bookworm
-
-RUN apt-get update && apt-get install -y cmake clang
-
+FROM rust:1.80-alpine3.19 as build 
+RUN apk add --no-cache git cmake clang libressl-dev
 WORKDIR /
 RUN git clone --depth 1 --branch v1.0.0-beta.4  https://github.com/dashevo/platform
-
 WORKDIR /app
 COPY Cargo.lock /app
 COPY Cargo.toml /app
 COPY migrations /app/migrations
 COPY src /app/src
-
-RUN cargo build
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --target x86_64-unknown-linux-musl --release
 RUN cargo install refinery_cli
+
+FROM scratch as indexer
+WORKDIR /app
+COPY --from=build /app/target/x86_64-unknown-linux-musl/release /app  
+CMD ["/app/indexer"]
```

Comment:
If you are changing the way image command executes, you should also reflect that in the CI deploy workflow

Reply:
✅